spec/active_data_tables_spec.rb

```diff
@@ -170,6 +170,15 @@
       subject { Event }
     end
 
+    it 'handles nil values when sorting a collection of hashes' do
```

Comment:
The test should be described in the `active_data_tables` shared_example definitions so it's tested against both ActiveRecord objects and data in an array. Follow the existing structure of the tests too.

You'll need to add a new nullable column to the existing dataset (the `Event` table is defined in `spec_helper`) with a mix of nil and assigned values.